int64[10] radius_array
---
float64[10] calc_array

